controllers/suite_test.go

```diff
@@ -138,7 +138,24 @@ var _ = Describe("Reconcile Test", func() {
 			// There should always be a "Ready" condition, regardless of Status.
 			conds := getOperator.Status.Conditions
 			Expect(conds).To(Not(BeEmpty()))
-			Expect(conds).To(ContainElement(HaveField("Type", operatorsv1alpha1.TypeReady)))
+			types := operatorsv1alpha1.GetTypes()
+			Expect(conds).To(HaveLen(len(types)))
+			for _, t := range types {
+				Expect(conds).To(ContainElement(HaveField("Type", t)))
```

Comment:
```suggestion
				Expect(meta.FindStatusCondition(conds, t)).To(Not(BeNil()))
```

Reply:
I can't find this API, as in
```
$ go get k8s.io/apimachinery/pkg/apis/meta
go: module k8s.io/apimachinery@upgrade found (v0.26.0), but does not contain package k8s.io/apimachinery/pkg/apis/meta
$ go install k8s.io/apimachinery/pkg/apis/meta@v0.26.0
go: k8s.io/apimachinery/pkg/apis/meta@v0.26.0: module k8s.io/apimachinery@v0.26.0 found, but does not contain package k8s.io/apimachinery/pkg/apis/meta
```
‽‽

Reply:
https://pkg.go.dev/k8s.io/apimachinery/pkg/api/meta#FindStatusCondition

Reply:
But I can't install it to actually run with it! (See errors in my previous comment)

Reply:
```
$ go mod tidy
go: finding module for package k8s.io/apimachinery/pkg/apis/meta
github.com/operator-framework/operator-controller/controllers tested by
        github.com/operator-framework/operator-controller/controllers.test imports
        k8s.io/apimachinery/pkg/apis/meta: module k8s.io/apimachinery@latest found (v0.26.0), but does not contain package k8s.io/apimachinery/pkg/apis/meta

Reply:
NVM... `apis` vs `api` _sigh_ *headdesk*